lib/march_hare/channel.rb

```diff
@@ -614,10 +614,14 @@ def basic_get(queue, auto_ack)
       end
     end
 
-    def basic_consume(queue, auto_ack, consumer)
```

Comment:
I don't think is it really a breaking change?

if you call it with 3 parameter the last one will be `consumer` - this is the same behaviour as before.
if you call it with 4 you have both `consumer_tag` and `consumer`